virtool/subtractions/api.py

```diff
@@ -60,14 +61,14 @@ async def find(req):
         async for document in db.subtraction.find({**db_query, **BASE_QUERY}, ["name", "ready"]):
             documents.append(base_processor(document))
 
-        return json_response(documents)
+        return json_response(await sort_alphabetically(documents))
```

Comment:
https://motor.readthedocs.io/en/stable/tutorial-asyncio.html#querying-for-more-than-one-document